Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: clickhouse/README.md

See the [datasets reference](https://docs.spiceai.org/reference/spicepod/datasets) for more dataset configuration options.

> Set the environment variable `CLICKHOUSE_PASS` to the password for your Clickhouse instance. This can be specified on the command line when running the Spice runtime, or in a `.env` file in the same directory as `spicepod.yaml`.

Reviewer: ```suggestion
Set the environment variable `CLICKHOUSE_PASS` to the Clickhouse instance password. Environment variables can be specified on the command line when running the Spice runtime or in a `.env` file in the same directory as `spicepod.yaml`.
```

i.e. to set the password in a `.env` file:

```bash

> echo "CLICKHOUSE_PASS=<password>" > .env

Reviewer: Provide example of resulting `.env` file.
